packages/office-addin-lint/src/commands.ts

```diff
@@ -25,7 +25,7 @@ export async function lint(command: commander.Command) {
   } catch (err) {
     // no need to display an error since there will already be error output;
     // just return a non-zero exit code
-    process.exitCode = defaults.ESLintExitCode.ToolingError;
+    process.exitCode = defaults.ESLintExitCode.CommandFailed;
```

Comment:
shouldn't this exit code reflect the command exit code?  Perhaps these should all just be `process.exitCode = err && typeof(err.status) == "number" ? err.status : ESLintExitCode.CommandFailed`. 

This is just that if you use the command and there are lint errors, the exit code should be 1 to reflect that.

Reply:
Exit code issue after manual testing has been fixed 👍